Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
```

Comment:
I would not assume anything regarding security. The classes themselves can't be executed from browser. However:

- We can recommend to disable PHP execution by adding .htaccess to /Tests 
```
<Files *.php>
deny from all
</Files>

```
- we can modify suggested Apache settings to add blocking like this. Sometime later we plan to work on an updated Apache config with a much tighter restriction. 

- generally, we need to move folders like  /Tests/ and /sql/ and perhaps some others from webserver Root.